src/index.html

```diff
@@ -36,7 +36,13 @@ <h1 class="heading">CROP AI</h1>
         </div>
       </div>
 
-      <div id="input_container" class="hidden"></div>
+      <div id="input_container" class="hidden">
+        <form action="/action_page.php" class="input_container_form">
+          <label for="files">Select files:</label>
+          <input type="file" id="files" name="files" multiple><br><br>
```

Comment:
It should be a single file input,
Design the form, make it a bit colorful, 
I would suggest using bootstrap, if you have worked on it earlier... it will make your work easy